### PR TITLE
Readme improvements + Roadmap page

### PR DESCRIPTION
**Pull request by tomas-quix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,12 +27,12 @@ Using Quix Streams, you can:
 
 - Produce time-series and event data to a Kafka Topic.
 - Consume time-series and event data from a Kafka Topic.
-- Process data by consuming it from one Kafka Topic, process it, and then producing the results back to another Kafka Topic.
-- Group data by Streams attaching metadata to them.
+- Process data by creating pipelines using <b>publish–subscribe</b> pattern.
+- Group data by streams to send different type of data (Timeseries data, events, metadata or binary) into one ordered stream of data.
```
Comment (PatrickMiraP) [inline README.md]: I would remove "data" that is in the brackets like this: "(Timeseries, events, metadata or binary)"
"data" after Timeseries sounds a bit redundant after saying "type of data ..."

File: README.md [hunk 2]
```diff
@@ -219,11 +219,11 @@ For full documentation of how to [<b>consume</b>](https://www.quix.io/docs/sdk/r
 
 This library provides several features and solves common problems you face when developing real-time streaming applications. 
 
-- <b>Stream context</b>: Quix Streams handles stream contexts for you, so all the data from one data source is bundled in the same scope. It let's you produce <b>multiple streams</b> through the same topic and enables the message broker to <b>parallelize loads</b> reliably across multiple consumers. 
+- <b>Stream context</b>: Quix Streams handles stream contexts for you, so all the data from one data source is bundled in the same scope. It let's you produce <b>multiple streams</b> through the same topic and enables the message broker to <b>parallelize loads</b> reliably across multiple consumers. It also allows stateful processing to be resiliently parallelised.
```
Comment (PatrickMiraP) [inline README.md:225]: About this part: "It also allows stateful processing to be resiliently parallelised." 
The library is not helping a lot in that regard yet. I would avoid that part until we implement what is in the roadmap.

File: roadmap.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Let's illustrate the problem on very standard example of stateful processing - rolling window of last 10 minutes data. If you perform this operation by yourself, you need to keep an eye on following:
+ - Management of state in memory
+   - Keep current rolling window in memory and remove old rows outside the window
+   - Append new rows and produce new output rows
+ - Kafka Checkpointing working hand to hand with state persistence to database
+ - Internal queue with incoming messages consumed by workers is synchronized with checkpointing 
+ - State recovery after service restart
+
+What we are working on is solving all of these problems in our QuixStreams library and give it familiar interface - Pandas DataFrames.
```
Comment (PatrickMiraP) [inline roadmap.md]: I would refactor the part of "our QuixStreams", it sounds a bit weird in an OS context. I would put this instead:
"What we are working on is solving all these common problems by adding some new features to the library, giving it a familiar interface in the vein of Pandas DataFrames."

**Review by PatrickMiraP — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tomas-quix) [reply README.md:225]: Actually, stream context does. Also, I have created this new helper class with is for stateful processing inside streams quite nice.

**Pull request closed without merging**